app/Services/ExchangeRateGetter.php

```diff
@@ -4,68 +4,93 @@
 class ExchangeRateGetter {
 	public function get() {
 		
```

Comment:
And here check is the `date` variable null and then act accordingly
```suggestion
		$date = $date ?? now()->toDateString();
```